src/wp-includes/blocks/calendar.php

```diff
@@ -24,10 +24,8 @@ function render_block_core_calendar( $attributes ) {
 			strpos( $permalink_structure, '%monthnum%' ) !== false &&
 			strpos( $permalink_structure, '%year%' ) !== false
 		) {
-			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited
 			$monthnum = $attributes['month'];
-			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited
-			$year = $attributes['year'];
+			$year     = $attributes['year'];
 		}
```

Comment:
The E2E tests are failing on the changes in this file.

I imagine this is due to the blocks code being maintained in a separate repo. In that case, the particular commit containing this change can be ignored.

Note: the `WordPress.WP.GlobalVariablesOverride` sniff is not active in WP Core as it is intended only for plugins and themes, so this ignore annotation really doesn't make sense (for Core).

Reply:
> The E2E tests are failing on the changes in this file.

Are you able to remove these changes from 26ce822d20259762c6205f951a0cdbc1fb8deb55 and force push, just to be certain all other tests are passing as expected? 

I'm sure they are but it's nice to be certain.

Reply:
Of course, not a problem. Rebased on current `master`, undid this particular change and force pushed.

Reply:
All 🟢